### ISSUE TITLE
termux:float not visible

### ISSUE THREAD

**Issue by fanoush** [termux/termux-float#12]
Hi. I installed 0.11 from f-droid on Xiaomi Redmi 4x (MIUI 8.2, Android 6.0.1) and the floating window is not shown. I can see notification with "Touch to show/hide window'  that changes when touched but  I don't see the window no matter what the notification says (hide or show). Any tip what is wrong and how to troubleshoot this?

BTW normal (non floating) termux works fine of course.

**Comment by fanoush** [comment]
Here is logcat | grep termux


toggle statusbar twice to show/hide

12-18 11:01:37.360  2152  2172 D StatusBar: onNotificationPosted:  Key: 0|com.termux.window|-559082697|null|10199 GroupKey: 0|com.termux.window|-559082697|null|10199 Connected: true
12-18 11:01:37.364  2152  2152 D PhoneStatusBar: updateNotification pkg=com.termux.window;id=-559082697
12-18 11:01:37.717  2152  2236 V PackageEntity: packageName = com.termuxdirtyClick = 0, dirtyShow = 0
12-18 11:01:37.723  2152  2236 V NotificationSortHelper: update package = com.termux.window click = 22, showCount = 2, id = 1022
12-18 11:01:37.724  2152  2236 V PackageEntity: packageName = com.termux.windowdirtyClick = 22, dirtyShow = 2


12-18 11:01:49.666  2152  3629 D StatusBar: onNotificationPosted:  Key: 0|com.termux.window|-559082697|null|10199 GroupKey: 0|com.termux.window|-559082697|null|10199 Connected: true
12-18 11:01:49.667  2152  2152 D PhoneStatusBar: updateNotification pkg=com.termux.window;id=-559082697
12-18 11:01:50.014  2152  2236 V PackageEntity: packageName = com.termuxdirtyClick = 0, dirtyShow = 0
12-18 11:01:50.019  2152  2236 V NotificationSortHelper: update package = com.termux.window click = 23, showCount = 2, id = 1022
12-18 11:01:50.020  2152  2236 V PackageEntity: packageName = com.termux.windowdirtyClick = 23, dirtyShow = 2

kill com.termux.window

12-18 11:02:06.379  4078  4078 D ApplicationsState: The current packageName is: com.termux.api  current info's userId is: 0
12-18 11:02:06.380  4078  4078 D ApplicationsState: The current packageName is: com.termux.boot  current info's userId is: 0
12-18 11:02:06.382  4078  4078 D ApplicationsState: The current packageName is: com.termux.widget  current info's userId is: 0
12-18 11:02:06.382  4078  4078 D ApplicationsState: The current packageName is: com.termux.window  current info's userId is: 0
12-18 11:02:06.383  4078  4078 D ApplicationsState: The current packageName is: com.termux  current info's userId is: 0
12-18 11:02:14.388  1735  2336 I ActivityManager: Force stopping com.termux.window appid=10199 user=0: from pid 4078
12-18 11:02:14.388  1735  2336 I ActivityManager: Killing 14170:com.termux.window/u0a199 (adj 2): stop com.termux.window
12-18 11:02:14.389  1735  2336 I AutoStartManagerService: MIUILOG- Reject RestartService packageName :com.termux.window uid : 10199
12-18 11:02:14.392  2152 22391 D StatusBar: onNotificationRemoved:  Key: 0|com.termux.window|-559082697|null|10199
12-18 11:02:14.397  2152  2152 D PhoneStatusBar: removeNotification key=android.os.Binder@2b5e191 keyCode=45474193 old=StatusBarNotification(pkg=com.termux.window user=UserHandle{0} id=-559082697 tag=nul
l score=-20 key=0|com.termux.window|-559082697|null|10199: Notification(pri=-2 contentView=com.termux.window/0x1090091 vibrate=null sound=null defaults=0x0 flags=0x62 color=0xff000000 vis=PRIVATE))
12-18 11:02:14.435  1735  1747 I WindowState: WIN DEATH: Window{652651e u0 com.termux.window}
12-18 11:02:14.435  1735  1747 W AppOps  : Finishing op nesting under-run: uid 10199 pkg com.termux.window code 24 time=0 duration=0 nesting=0

start again, I see toast message about resizing window but no floating windows opens


12-18 11:02:29.210  1735  2335 I ActivityManager: START u0 {act=android.intent.action.MAIN cat=[android.intent.category.LAUNCHER] flg=0x10200000 cmp=com.termux.window/.TermuxFloatActivity (has extras)} f
rom uid 10155 on display 0
12-18 11:02:29.249  1735  2467 I ActivityManager: Start proc 14289:com.termux.window/u0a199 for activity com.termux.window/.TermuxFloatActivity
12-18 11:02:29.283  2420  2543 D WtProcessController: set foreground process size 1 pid:14289pacakgeName:com.termux.window
12-18 11:02:29.356  1735  2469 I PackageManager: com.termux.api mDexOptPerformed=[Ljava.lang.Object;@92b6a14
12-18 11:02:29.357  1735  2469 I PackageManager: com.termux.boot mDexOptPerformed=[Ljava.lang.Object;@d62b186
12-18 11:02:29.359  1735  2469 I PackageManager: com.termux.widget mDexOptPerformed=[Ljava.lang.Object;@c5673e7
12-18 11:02:29.359  1735  2469 I PackageManager: com.termux.window mDexOptPerformed=[Ljava.lang.Object;@3c9eb94
12-18 11:02:29.362  1735  2469 I PackageManager: com.termux mDexOptPerformed=[Ljava.lang.Object;@a587284
12-18 11:02:29.553  2152  4049 D StatusBar: onNotificationPosted:  Key: 0|com.termux.window|-559082697|null|10199 GroupKey: 0|com.termux.window|-559082697|null|10199 Connected: true
12-18 11:02:29.554  2152  2152 D PhoneStatusBar: addNotification pkg=com.termux.window;basepkg=com.termux.window;id=-559082697
12-18 11:02:31.174  1735  2335 D PerfShielderService: com.termux.window|Toast|88|529148989686753|86|3

**Comment by fanoush** [comment]
OK, I found it. It is security settings issue. This article helped me http://en.miui.com/thread-708221-1-1.html In MIUI there is separate setting for app permissions, app notifications and then there is special security settings which again lists applications and allows you to enable disable yet another set of permissions and there I found it - something like 'allow popups' was disabled. The screens in that linked article were a lot different and it did not ask me first time when I started it but I found it anyway. Now it works So I am closing this.
